setup.py

```diff
@@ -58,6 +58,7 @@
         'aioredis>=1.1.0,<2.0.0',
         'click>=6.7',
         'pydantic>=1',
+        'pytz',
```

Comment:
No need to add pytz as a dependency, post people won't use it.

And if you did make it a dependency, you'd need to restrict it.